docs/supervisor_and_streams.rst

```diff
@@ -84,6 +113,7 @@ Finally, restart supervisor:
 
 Streamsample setup
 ------------------
+
 A template streamsample configuration file "streamsample.conf.template" is
 included in the SFM distribution.  To set up a streamsample process managed by
 supervisor:
```

Comment:
@kerchner On Line 129 below, instead of mentioning your preferred system user account we can just say the user you created in above steps or put www-data in brackets to have clarity.